src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 0 16px;
+  width: 200px;
+  box-sizing: border-box;
+  display: block;
+  background-color: $bodyColor;
+  border-radius: 5px;
+  border: 1px solid $whiteGrayColor;
+}
+
+.card__img {
+  margin-left: 19px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__info {
+  height: 36px;
+  width: 166px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $nameTextColor;
+  margin-top: 36px;
+  margin-left: 16px;
+  margin-right: 16px;
+  h3 {
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+  p {
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+  }
```

Comment:
- As it's a `scss` you can use nesting. Example:
```scss
.card {
  // card styles

  &_img {
    // image styles
  }
}
```

- Don't use tag names for styling (except html and body)
<img width="864" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/4216f4f9-04fb-42a3-b8b3-ce7b2050f155">

Fix in all places